File: MyWebSite.Server/Data/Entities/CV.cs
using Microsoft.CodeAnalysis.Host;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json;
using System.Text.Json.Serialization;

namespace MyWebSite.Server.Data.Entities
{
    public class CV
    {
        public Guid Id { get; set; }
        public string Picture { get; set; }
        public string FullName { get; set; }
        public bool IsMale { get; set; }
        public string Nationality { get; set; }
        public DateTime BirthDate { get; set; }
        public string Address { get; set; }
        public string Profession { get; set; }
        public string Description { get; set; }
        public ICollection<Contact>? Contacts { get; set; }
        public ICollection<Skill>? Skills { get; set; }
        public ICollection<WorkExperience>? WorkExperience { get; set; }
        public ICollection<Education>? Education { get; set; }
        public ICollection<Language>? Languages { get; set; }
        public ICollection<Certificate>? Certificates { get; set; }
    }
}

File: mywebsite.client/src/app/api/models/cv.ts
/* tslint:disable */
/* eslint-disable */
import { Certificate } from '../models/certificate';
import { Contact } from '../models/contact';
import { Education } from '../models/education';
import { Language } from '../models/language';
import { Skill } from '../models/skill';
import { WorkExperience } from '../models/work-experience';
export interface Cv {
  address?: string | null;
  birthDate?: string;
  certificates?: Array<Certificate> | null;
  contacts?: Array<Contact> | null;
  description?: string | null;
  education?: Array<Education> | null;
  fullName?: string | null;
  id?: string;
  isMale?: boolean;
  languages?: Array<Language> | null;
  nationality?: string | null;
  picture?: string | null;
  profession?: string | null;
  skills?: Array<Skill> | null;
  workExperience?: Array<WorkExperience> | null;
}
